projects/common/src/time/time.ts

```diff
@@ -32,11 +32,29 @@ export class Time {
     return this._date;
   }
 
+  public static parse(label: TimeString, isUTC: boolean = false): Time {
+    const time: [string, Meridian] = label.split(' ') as [string, Meridian];
+    const meridian: Meridian = time[1];
+    const [hours, minutes, seconds, milliseconds] = time[0].split(':').map(value => +value);
```

Comment:
do not do implicit conversion using `+`. Use numberCoercer instead.

Reply:
got it, saw it being used somewhere else also, that's why I thought it would be fine.

Reply:
Updated